src/OrchardCore.Modules/OrchardCore.Email/Manifest.cs

```diff
@@ -6,6 +6,5 @@
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
     Description = "Provides email settings configuration and a default email service based on SMTP.",
-    Dependencies = new[] { "OrchardCore.Resources" },
```

Comment:
The module [uses CodeMirror](https://github.com/OrchardCMS/OrchardCore/blob/ae2f9525e29f533b1a993d7279c9a45d06ac5198/src/OrchardCore.Modules/OrchardCore.Email/Views/Items/EmailTask.Fields.Edit.cshtml#L67) from the Resources module, so the dependency is warranted, even if in most scenarios, due to the Resources module being enabled anyway, this wouldn't cause an issue.